api/src/test/java/org/motechproject/nms/api/osgi/UserControllerBundleIT.java

```diff
@@ -67,9 +117,109 @@ private void setupData() {
         Subscription subscription3 = subscriptionDataService.create(new Subscription("003", subscriber2, pack2));
     }
 
+    private void createFLWCappedServiceNoUsageNoLocationNoLanguage() {
+        cleanAllData();
+
+        FrontLineWorker flw = new FrontLineWorker("Frank Llyod Wright", "0000000000");
+        frontLineWorkerService.add(flw);
+
+        Language language = new Language("Papiamento", 99);
+        languageDataService.create(language);
+
+        CircleLanguage circleLanguage = new CircleLanguage("AA", language);
+        circleLanguageDataService.create(circleLanguage);
+
+        ServiceUsageCap serviceUsageCap = new ServiceUsageCap(null, Service.MOBILE_KUNJI, 3600);
+        serviceUsageCapDataService.create(serviceUsageCap);
+    }
+
+    private void createFLWWithLanguageServiceUsageAndCappedService() {
+        cleanAllData();
+
+        Language language = new Language("English", 10);
+        languageDataService.create(language);
+
+        FrontLineWorker flw = new FrontLineWorker("Frank Llyod Wright", "0000000000");
```

Comment:
I'm thinking of a different F. L. W. :)

On Mon, Apr 20, 2015 at 12:47 PM, Lauren Lavoie <notifications@github.com>
wrote:

> In
> api/src/test/java/org/motechproject/nms/api/osgi/UserControllerBundleIT.java
> <https://github.com/koshalt/mim/pull/46#discussion_r28722137>:
>
> > +        languageDataService.create(language);
> > +
> > +        CircleLanguage circleLanguage = new CircleLanguage("AA", language);
> > +        circleLanguageDataService.create(circleLanguage);
> > +
> > +        ServiceUsageCap serviceUsageCap = new ServiceUsageCap(null, Service.MOBILE_KUNJI, 3600);
> > +        serviceUsageCapDataService.create(serviceUsageCap);
> > +    }
> > +
> > +    private void createFLWWithLanguageServiceUsageAndCappedService() {
> > +        cleanAllData();
> > +
> > +        Language language = new Language("English", 10);
> > +        languageDataService.create(language);
> > +
> > +        FrontLineWorker flw = new FrontLineWorker("Frank Llyod Wright", "0000000000");
>
> Also, you misspelled Lloyd :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/koshalt/mim/pull/46/files#r28722137>.
>